example/configs/wasm.json

```diff
@@ -0,0 +1,35 @@
+{
+  "type": "wasm",
+  "value": {
+    "chain-id": "constantine-3",
+    "nid": "1",
```

Comment:
this should not be required

Reply:
@izyak Actually, we do need NID because it will be different for different chains. Moreover, this wasm module is built to support almost all cosmwasm chains.